projects/table-builder/src/lib/components/filter/filter.component.html

```diff
@@ -3,23 +3,23 @@
     <input type="hidden" name="filterId" [ngModel]="filter.filterId" />
     <input type="hidden" name="key" [ngModel]="filter.key" />
     <input type="hidden" name="fieldType" [ngModel]="filter.fieldType" />
-    <div class="row">
+    <div class="row" >
       <h4 class="filter-name">{{(filter.key | spaceCase)}} Filter</h4>
       <button class="cancel-button" color="primary" mat-icon-button (click)="close.emit();" type="button"
           matTooltip="Close">
         <mat-icon class="cancel-button" color="primary">close</mat-icon>
       </button>
     </div>
     <div class="row">
-        <mat-form-field class="my-filter">
+        <mat-form-field class="my-filter" *ngIf='filter.fieldType !== FieldType.Enum' >
```

Comment:
I assume this is because there is only one filter type. But what about null check (i.e is blank)

Reply:
I don't believe that's possible. in any case it's unclear why showing this filter would be correct for nulls

Reply:
Isn't this the list of all available filters?

Reply:
Done.